examples/platform/nxp/common/app_task/source/AppTaskBase.cpp

```diff
@@ -89,20 +89,15 @@
 #include "OTARequestorInitiator.h"
 #endif
 
-#if CONFIG_CHIP_TEST_EVENT && CHIP_DEVICE_CONFIG_ENABLE_OTA_REQUESTOR
-#include <app/clusters/ota-requestor/OTATestEventTriggerDelegate.h>
+#if CONFIG_CHIP_REGISTER_SIMPLE_TEST_EVENT_TRIGGER_DELEGATE && CHIP_DEVICE_CONFIG_ENABLE_OTA_REQUESTOR
```

Comment:
Can you use #ifdef CONFIG_CHIP_REGISTER_SIMPLE_TEST_EVENT_TRIGGER_DELEGATE instead of #if as CONFIG_CHIP_REGISTER_SIMPLE_TEST_EVENT_TRIGGER_DELEGATE is a boolean config value.